File: src/main/java/no/nav/domain/pensjon/regler/vedtak/VilkarsprovAlderspensjonResultat.java
package no.nav.domain.pensjon.regler.vedtak;

import no.nav.domain.pensjon.regler.beregning2011.AbstraktBeregningsResultat;

import java.lang.reflect.Constructor;
import java.lang.reflect.InvocationTargetException;

public class VilkarsprovAlderspensjonResultat extends AbstraktVilkarsprovResultat {


    private AbstraktBeregningsResultat beregningVedUttak;
    private VilkarsprovInformasjon vilkarsprovInformasjon;

}

File: src/main/java/no/nav/pensjon/regler/domain/vedtak/VilkarsprovAlderspensjonResultat.kt
package no.nav.pensjon.regler.domain.vedtak

import no.nav.pensjon.regler.domain.beregning2011.AbstraktBeregningsResultat

class VilkarsprovAlderspensjonResultat : AbstraktVilkarsprovResultat() {
    var beregningVedUttak: AbstraktBeregningsResultat? = null
    var vilkarsprovInformasjon: VilkarsprovInformasjon? = null
}